.gitignore

```diff
@@ -4,3 +4,4 @@ build/
 .idea/
 src/index.php
 src/Mocks
+tests/build
```

Comment:
Why was this added? When is this directory created? It does not seem to be related to your change.